server/task_check_executor_statement_advisor_composite.go

```diff
@@ -46,7 +46,7 @@ func (exec *TaskCheckStatementAdvisorCompositeExecutor) Run(ctx context.Context,
 			return []api.TaskCheckResult{
 				{
 					Status:  api.TaskCheckStatusWarn,
-					Code:    common.TaskCheckEmptySchemaReviewPolicy,
+					Code:    api.ConvertToErrorCode(advisor.EmptySchemaReviewPolicy),
```

Comment:
Since we are testing the server logic, I feel we should use the common error code directly. In general, we should use the same domain. If writing tests under advisor module, we use the advisor error code, if under server module, then use the server error code 

Otherwise, we can't catch the logic conversion error in api.ConvertToErrorCode. e.g. we have typo like converting advisor.errorFoo to common.errorBar

same below